### DISCUSSION TITLE
can't build on iOS

### DISCUSSION THREAD

**Discussion by RafaelR7** [firebase/flutterfire#3805] [Q&A] [answered]
I'm on flutter stable 1.22.0 and had to upgrade most of the flutterFire packages, trying to build the app, I got an error saying to run `pod repo update`, run the command, but still receive the same error when building the app, I tried `pod install` and `pod update` but got the errors
<img width="759" alt="Screen Shot 2020-10-07 at 13 44 15" src="https://user-images.githubusercontent.com/20539985/95361550-4083c100-08a3-11eb-8c6b-38893f434acc.png">

<img width="763" alt="Screen Shot 2020-10-07 at 13 46 10" src="https://user-images.githubusercontent.com/20539985/95361721-79bc3100-08a3-11eb-887e-6b6360560df7.png">

tried the commands: `pod install --repo-update` and `pod update Firebase/Storage` but still got the same error message

I need a help, how can I solve these errors?

**Comment by thubv** [comment]
try create new flutter project, copy and overwrite from ios/Flutter to ios/Flutter of old project

**Comment by kasyrm** [comment, marked as answer]
Faced the same issue yesterday. After removing podfile.lock, derived data and running flutter clean it started working.
